meson.build

```diff
@@ -174,3 +174,14 @@ endif
 if get_option('tests')
   subdir('tests')
 endif
+
+message()
+message('   @0@ @1@'.format(meson.project_name(), meson.project_version()))
+message('   ===================')
+message()
+message('   man pages (xsltproc):                        @0@'.format(build_man_page))
+message('   SELinux:                                     @0@'.format(cdata.has('HAVE_SELINUX')))
+message('    ... >= 2.3:                                 @0@'.format(cdata.has('HAVE_SELINUX_2_3')))
+message('   require default userns:                      @0@'.format(get_option('require_userns')))
+message('   mysteriously satisfying to pop:              true')
+message()
```

Comment:
If the concept of "put a summary at the end" gets propagated from Autotools into Meson, then it should be done by using Meson's `summary()`:

```
if meson.version().version_compare('>=0.53')
  summary(...)
endif
```

But, again, if we do that, it should be limited to only the information that is genuinely interesting, otherwise it will defeat the purpose of having a summary at the end.